### ISSUE TITLE
MATLAB crash when saving clusters

### ISSUE THREAD

**Issue by perczelgy** [csn-le/wave_clus#90]
Hi Fernando,

I am experiencing the following error: I do the spike sorting using wave_clus 3.0 in batch mode, and to a little bit of manual editing finally using the GUI. However, when I try to save the cluster using the 'Save clusters' button, MATLAB suddenly crashes with a message in the command window: 
Warning: UI controls may not print or export properly because the output has been scaled. Setting the
figure's PaperPositionMode property to 'auto' may help. 
... and a pop-up window:
![kep](https://user-images.githubusercontent.com/26854636/44735675-64358400-aaed-11e8-9708-fdcd3b505394.png)

Could you please help me how to fix this bug?

Thank you in advance: György

**Comment by ferchaure** [comment]
Which Matlab version are you using?

**Comment by perczelgy** [comment]
I am sorry to hear that. Currently I am using MATLAB R2015a. Can you test it with that version? Otherwise could you recommend a version that fully supports wave_clus 3.0? (I did not have any problem with this function when using previous versions of wave_clus.)

**Comment by ferchaure** [comment]
I always check with 2012 and 2017a. 
That error is happening always?

**Comment by perczelgy** [comment]
Unfortunately yes.

**Comment by ferchaure** [comment]
How many clusters do you have?
It's quite odd, for now you can comment the line 599 in wave_clus.m and don't save the new figure as a image.

**Comment by perczelgy** [comment]
I have from 2-3 to 8-9 clusters, but the problem occurs every time. 
You mean lines 599-601? 
![kep](https://user-images.githubusercontent.com/26854636/44842342-df11b280-ac45-11e8-9a57-fcee697cc5d8.png)
I tried that, but unfortunately MATLAB still crashes.

**Comment by ferchaure** [comment]
sorry, line 597.
but before that, try:
remove the second part of the line 596 (after the first ;)
remove the '-r300' argument from the line 597

**Comment by ferchaure** [comment]
any luck with this issue?

**Comment by perczelgy** [comment]
Sorry for the late response, I did not have the time, to try it out until now.
Tried the first version: 
 "remove the second part of the line 596 (after the first ;)
  remove the '-r300' argument from the line 597"
... and it did work. 
Thank you very much!
(Just a comment: I had to try wave_clus on MATLAB R2014b and experienced the same issue - though I suppose the above instructions should solve the problem there too).
